lighthouse-core/fraggle-rock/user-flow.js

```diff
@@ -20,6 +22,8 @@ class UserFlow {
   constructor(page, options) {
     /** @type {FrOptions} */
     this.options = {page, ...options};
+    /** @type {string} */
+    this.name = (options && options.name) || DEFAULT_FLOW_NAME;
```

Comment:
WDYT of deriving this from the origin visited at the end by default?

Reply:
SGTM, why the end and not the start though?

Reply:
> why the end and not the start though?

We won't know what they decide to actually measure until they invoke `navigation/snapshot/timespan` i.e. `page`'s current URL could be not what the flow is